db/schema.rb

```diff
@@ -46,21 +46,11 @@
 
   create_table "users", force: :cascade do |t|
     t.string "name"
-    t.string "photo"
+    t.text "photo"
     t.text "bio"
-    t.integer "post_counter", default: 0
+    t.integer "post_counter"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.string "email", default: "", null: false
-    t.string "encrypted_password", default: "", null: false
-    t.string "reset_password_token"
-    t.datetime "reset_password_sent_at"
-    t.datetime "remember_created_at"
-    t.string "confirmation_token"
-    t.datetime "confirmed_at"
-    t.datetime "confirmation_sent_at"
-    t.string "unconfirmed_email"
-    t.string "role", default: "default"
```

Comment:
- [ Required ] I think one of the reasons why your devise views have an error is that you might have removed the above highlighted `fields` from your user's table which are critical to the proper functioning of the devise gem. Please consider setting up the devise gem step by step by following this tutorial [getting started link >>](https://github.com/heartcombo/devise)